package.json

```diff
@@ -29,7 +29,7 @@
   "scripts": {
     "start": "DEBUG=raneto node example/server.js",
     "start_win": "set DEBUG=raneto&&node example/server.js",
-    "test": "npm run lint & npm run compile & mocha --reporter spec test/*.js",
+    "test": "npm run lint && npm run compile && mocha --reporter spec test/*.js",
```

Comment:
yea I did have problems with this section where the test runner actually depends on the code being successfully compiled first. using `&&` here ensures the following happens sequentially (rather than in parallel as `&` implies): lint, compile, test. if any one of these steps fail, the following steps do not run. let me know if you have other questions about this, I'm happy to answer them.